internal/commands/cmdautovc.go

```diff
@@ -0,0 +1,146 @@
+package commands
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/zekroTJA/shinpuru/internal/services/database"
+	"github.com/zekroTJA/shinpuru/internal/util"
+	"github.com/zekroTJA/shinpuru/internal/util/static"
+	"github.com/zekroTJA/shinpuru/pkg/acceptmsg"
+	"github.com/zekroTJA/shinpuru/pkg/fetch"
+	"github.com/zekroTJA/shinpuru/pkg/stringutil"
+	"github.com/zekroTJA/shireikan"
+	"github.com/zekrotja/dgrs"
+)
+
+type CmdAutovc struct {
```

Comment:
Awesome that you also provided an implementation for the legacy command system, but you forgot to register the command in the handler. ^^

Just add the following line to `internal/inits/legacycommandhandler.go`
```
cmdHandler.RegisterCommand(&commands.CmdAutovc{})
```